index.html

```diff
@@ -44,27 +44,27 @@ <h1>Codified Conduct</h1>
             {
               title: '"No Ideologies"',
               url: 'https://github.com/CodifiedConduct/coc-no-ideologies',
-              description: ''
+              description: 'A code of conduct focused on the restriction of discussion to technical topics, and disallowing discussions pertaining to non-technical ideologies.'
             },
             {
               title: 'Django',
               url: 'https://www.djangoproject.com/conduct',
-              description: ''
+              description: 'This code of conduct has six basic commandments: "Be friendly and patient", "Be welcoming", "Be considerate", "Be respectful", "Be careful in the words you choose", and "Try to understand disagreements".'
             },
             {
               title: 'Citizen',
               url: 'http://citizencodeofconduct.org',
-              description: ''
+              description: 'The website is temporarily down. Updates pending.'
             },
             {
               title: 'JavaScript Foundation',
               url: 'https://js.foundation/community/code-of-conduct',
-              description: ''
+              description: 'The summary of this CoC is to be a decent person and anything offensive that would not be tolerated at a workplace environment will not be tolerated in the community. '
             },
             {
               title: 'Contributor Covenant',
               url: 'https://www.contributor-covenant.org',
-              description: ''
+              description: 'Some open source projects attract enough contributors that a community forms. A healthy open source community centers the shared values and norms of its members. While not all of these values are the exactly the same from community to community, there is a set of core values and norms that are essential in a just and equitable software commons. Contributor Covenant is a code of conduct that you can adapt to express both these fundamental shared values, and the special norms and values that distinguish your own community.'
```

Comment:
Too long. No clue who this is for. The point of Codified Conduct is to give a quick, to-the-point, summary of CoC's.